### ISSUE TITLE
[New feature - optimization] delete all synchronised files

### ISSUE THREAD

**Issue by togglecode** [nextcloud/ios#169]
I was wondering if it's possible to have a feature under optimization which deletes all the synchronised images and videos from the device. The library [reference](https://developer.apple.com/reference/photos/phassetcollectionchangerequest?language=ob  ) says that this can be done after ios 8.

It would also be nice that this could be extended also for all the future image/video uploads as well for example this can be a scenario:
1. User enables option "delete all synchronised image/videos
2. App deletes all the files locally after user gives the permission
3.  User tries to upload 10 new picture (or takes 10 pictures with camera and "automatic camera upload is enabled)
4. App uploads all the pictures and at the end asks for permission to delete them

**Comment by togglecode** [comment]
Oh well, it seems this has been already mentioned in #157 as well, but I think it would be also great if all the previous files also get deleted and not only camera uploads.

**Comment by marinofaggiana** [comment]
This feature required the confirm deletion first of user, this is unacceptable with Auto Upload function.

**Comment by togglecode** [comment]
True, but a feature like "Delete files from storage" same like "delete files from cache" would be nice, which deletes all the synchronized files.
